src/cromshell/utilities/VariableSetup.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+
```

Comment:
docstring.

And I'd suggest giving it a better name, as `Variable` is too general.

Reply:
Done, renamed to CromshellConfig